lib/screens/downloads_screen.dart

```diff
@@ -36,8 +36,16 @@ class DownloadsScreen extends StatelessWidget {
               const Divider(),
             ]),
           ),
-          const DownloadedItemsList(),
-          // CurrentDownloadsList(),
+          DownloadedItemsTitle(title: AppLocalizations.of(context)!.artists),
+          const DownloadedItemsList(type: DownloadsScreenItemType.artists),
+          DownloadedItemsTitle(title: AppLocalizations.of(context)!.albums),
+          const DownloadedItemsList(type: DownloadsScreenItemType.albums),
+          DownloadedItemsTitle(title: AppLocalizations.of(context)!.playlists),
+          const DownloadedItemsList(type: DownloadsScreenItemType.playlists),
+          DownloadedItemsTitle(title: AppLocalizations.of(context)!.genres),
+          const DownloadedItemsList(type: DownloadsScreenItemType.genres),
+          DownloadedItemsTitle(title: AppLocalizations.of(context)!.tracks),
+          const DownloadedItemsList(type: DownloadsScreenItemType.tracks),
```

Comment:
The repetitions are a bit awkward, but we need to have all categories as direct sliver children of the scroll view.

Reply:
That's fine for now. The redesign of the downloads screen is still hazy, not sure if I want to go for tabs, screens, or filter chips for the different categories. So this is still subject to change :)

Reply:
I was actually thinking about filter chips earlier. But for the first iteration, I wanted to keep things simple.